packages/console/src/ConsoleMenu.tsx

```diff
@@ -188,6 +188,7 @@ class ConsoleMenu extends PureComponent<ConsoleMenuProps, ConsoleMenuState> {
     return (
       <>
         <Button
+          id="table-actions"
```

Comment:
We should not add an `id` to these elements, especially for tests. IDs are bad because they must be unique to that elements subtree and also do not provide any accessibility. It would be better to use a `data-testid` instead for it to be accessible with tests, _but_ we can actually do better. Take a look at the Locators page: https://playwright.dev/docs/locators
First preference is to `getByRole`, then `getByTest`, and `getByLabel`. We should be able to use `getByRole` here because we are already providing a tooltip on the button. When you provide a tooltip on an `iconOnly` button, we automatically add it as the `aria-label`: https://github.com/deephaven/web-client-ui/blob/e90b627fed2c76b81ee96006abef14942388f3cc/packages/components/src/Button.tsx#L137

So in this case, we shouldn't need to add an ID. You should be able to just do `getByRole('button', { name: 'Tables' })` and this should work.
Ditto with the Widgets button below.

Reply:
I noticed that the condition to set the tooltip as the `aria-label` isn't met since `iconOnly = false`.

https://github.com/deephaven/web-client-ui/blob/e90b627fed2c76b81ee96006abef14942388f3cc/packages/components/src/Button.tsx#L126-L127

Reply:
Ahhh right because we have the dropdown menu as a child. Whoops.
Add an `aria-label` explicitly then instead of `id`.